backend/app/assets/javascripts/spree/backend/orders/edit.js

```diff
@@ -2,4 +2,12 @@ $(document).ready(function () {
   'use strict';
 
   $('[data-hook="add_product_name"]').find('.variant_autocomplete').variantAutocomplete();
+
+  // Will refresh the states (based on the country) when you open the
+  // shipment address form
+  $(".js-new-shipment-address").click(function(){
+    var data_target = $(this).data("target");
```

Comment:
Identifier 'data_target' is not in camel case.<br>Mixed double and single quotes.